src/Illuminate/Foundation/helpers.php

```diff
@@ -456,8 +456,8 @@ function env($key, $default = null)
                 return;
         }
 
-        if (strpos($value, 'base64:') === 0) {
-            return base64_decode(substr($value, 7));
+        if (Str::startsWith($value, ['base64:', "'base64:", '"base64:'])) {
+            return base64_decode(trim($value, '"\''));
```

Comment:
Thanks. I've pushed that substr addition. Also, I noticed we don't have any tests for this `env` helper but I'm unsure how to mock that getenv call so I didn't add any. :(